plugins/inputs/procstat/procstat.go

```diff
@@ -93,6 +93,7 @@ func (p *Procstat) Gather(acc telegraf.Accumulator) error {
 		case "pgrep":
 			p.createPIDFinder = NewPgrep
 		default:
+			p.PidFinder = "pgrep"
```

Comment:
Does this fit better in the init function, as it seems to be a standard location for object defaults?

Reply:
I tried this out, it seems good but then it moves it away from the setting of `p.createPIDFinder`, which it would need updated alongside of.  We could set that in the init function too, but then we would need to introduce a initialized boolean or such.  We can rework this when we add constructor functions for the plugins (part of the config prototype).